_includes/callouts.html

```diff
@@ -5,35 +5,78 @@
             <div class="container">
                 <div class="columns is-multiline is-centered">
                     {% for callout in callouts.items %}
-                        <div class="column is-4 has-text-centered">
-                            {% if callout.icon %}
-                            <div class="icon callout-icon">
+                    <div class="column is-4 has-text-centered">
+                        {% if callout.icon %}
+                        <div class="icon callout-icon">
                             {% if callout.icon_brand %}
-                                <i class="fab {{ callout.icon }} fa-4x"></i>
+                            <i class="fab {{ callout.icon }} fa-4x"></i>
                             {% else %}
-                                <i class="fas {{ callout.icon }} fa-4x"></i>
-                            {% endif %}
-                            </div>
-                            {% endif %}
-                            <p class="title is-5">{{ callout.title }}</p>
-                            <p class="subtitle is-5">{{ callout.subtitle }}</p>
-                            
-                            {% if callout.description %}
-                            <div class="content">
-                                <p>{{ callout.description | newline_to_br }}</p>
-                            </div>
+                            <i class="fas {{ callout.icon }} fa-4x"></i>
                             {% endif %}
+                        </div>
+                        {% endif %}
+                        <p class="title is-5">
+                            <span id="countup_{{ forloop.index }}" data-final-count="{{ callout.title | remove: ',' }}">{{ callout.title }}</span>
+                        </p>
+                        <p class="subtitle is-5">{{ callout.subtitle }}</p>
 
-                            {% if callout.call_to_action_name %}
-                            <a href="{{ callout.call_to_action_link | relative_url }}" class="button is-primary">
-                                {{ callout.call_to_action_name }}
-                            </a>
-                            {% endif %}
+                        {% if callout.description %}
+                        <div class="content">
+                            <p>{{ callout.description | newline_to_br }}</p>
                         </div>
+                        {% endif %}
+
+                        {% if callout.call_to_action_name %}
+                        <a href="{{ callout.call_to_action_link | relative_url }}" class="button is-primary">
+                            {{ callout.call_to_action_name }}
+                        </a>
+                        {% endif %}
+                    </div>
                     {% endfor %}
                 </div>
             </div>
         </div>
     </section>
+
+<script>
```

Comment:
Can this be moved to a JS file and included in this HTML file?

Reply:
https://github.com/FRCTeam1987/teambroncobots.com/commit/d68eb5fcb5a2d595564218a092be624c81db6a2b